### DISCUSSION TITLE
LXQt  1.0.0 release: news collection

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2117] [Media]
I was thinking about collection some media coverage regarding 1.0.0 somewhere. Thoughts?

Examples:
* https://www.debugpoint.com/2021/11/lxqt-1-0-release/
* https://9to5linux.com/lxqt-1-0-0-desktop-environment-released-after-8-years-of-development-this-is-whats-ne
 EDIT: moved to "Media".

**Comment by edwardp3** [comment]
https://www.phoronix.com/scan.php?page=news_item&px=LXQt-1.0-Released

(Has quite a few comments...)

**Reply by tsujan** [reply]
> https://www.phoronix.com/scan.php?page=news_item&px=LXQt-1.0-Released

The comments at that site are mostly useless, at best. I don't know why.

EDIT: I mean any comment on any post; not just this one.

**Reply by edwardp3** [reply]
Understood.  :)

Here's another I found:

https://www.linuxtoday.com/news/lxqt-1-0-graphics-environment-released/

**Reply by tsujan** [reply]
> Understood. :)

Do you know why it's so? Most things phoronix announces are followed by irrelevant comments, as far as I've seen.

**Reply by edwardp3** [reply]
I replied "Understood", because I understood your reply. :)

I have noticed the same regarding the comments posted there. There are more well-known, well-established sites for tech news.

**Comment by stefonarch** [comment]
Another more interesting: https://www.theregister.com/2021/11/09/new_qt_linux_desktops/

But my initial idea wasn't maybe formulated well enough: we have Q&A, General and so on as categories here in _Discussions,_ - I was thinking of one only for media links, youtube tutorials ecc. (and discussing them maybe)

**Reply by tsujan** [reply]
> I was thinking of one only for media links, youtube tutorials ecc. (and discussing them maybe)

No objection from me.
